javascript/multiple-devices-test.js

```diff
@@ -0,0 +1,69 @@
+import 'babel-polyfill'
+import {assert} from 'chai'
+import Promise from 'bluebird'
+import parallel from 'mocha.parallel'
+import webdriver from 'selenium-webdriver'
+
+const By = webdriver.By
+const until = webdriver.until
+
+const kobitonServerUrl = 'https://api.kobiton.com/wd/hub'
+const username = 'your Kobiton username'
+const apiKey = 'your Kobiton api key'
+
+const desiredCapabilities = [{
+  sessionName:        'Automation test session on first device',
+  sessionDescription: 'This is an example for Android web', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  browserName:        'chrome', 
+  deviceGroup:        'KOBITON'
+  deviceName:         'Galaxy S5',
+  platformName:       'Android'
+  }, {
+  sessionName:        'Automation test session on second device',
+  sessionDescription: 'This is an example for Android web', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  browserName:        'chrome', 
+  deviceGroup:        'KOBITON', 
+  deviceName:         'Nexus 5',
+  platformName:       'Android'
+}]
+
+const buildDriver = (caps) => {
+  return new Promise((resolve, reject) => {
+    caps.username = username
+    caps.accessKey = apiKey
+
+    const driver = new webdriver.Builder()
+      .usingServer(kobitonServerUrl)
+      .withCapabilities(caps)
+      .build()
+    
+    resolve(driver)
+  })
+}
+
+parallel('Tests',() => {
+  desiredCapabilities.forEach((caps) => {
```

Comment:
Why not just

```
parallel("", () => {
  it("", () => {
    desiredCapabilities[0]
  })

  it("", () => {
    desiredCapabilities[1]
  })
})
```

Reply:
It will be duplicated codes in test cases

```
          driver.get('http://www.google.com/ncr')
          driver.findElement(By.name('q')).sendKeys('Kobiton')
          driver.findElement(By.name('btnG')).click()
          driver.getTitle().then((title) => {
            assert.include(title, 'Kobiton')
          })
```